pkg/cgroup/errors.go

```diff
@@ -0,0 +1,21 @@
+package cgroup
+
+import "fmt"
+
+type VersionNotSupported struct{}
```

Comment:
Because of:

![image](https://user-images.githubusercontent.com/7395852/203098019-cca34902-3f30-4680-8ae2-af9fb4b36e52.png)

I'm actually checking the type of the error to take decisions (unsupported version is an accepted error).